client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleMaster.java

```diff
@@ -102,6 +102,7 @@ public void registerJob(JobShuffleContext context) {
           celebornAppId = FlinkUtils.toCelebornAppId(lifecycleManagerTimestamp, jobID);
           LOG.info("CelebornAppId: {}", celebornAppId);
           lifecycleManager = new LifecycleManager(celebornAppId, conf);
+          lifecycleManager.applicationCount().increment();
```

Comment:
The application count should be recorded on a per-job basis rather than per cluster.